tests/daemonize/main.c

```diff
@@ -0,0 +1,46 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "daemonize.h"
+#include "warnp.h"
+
+int
+main(int argc, char * argv[])
+{
+	const char * pidfile;
+
+	/*
+	 * This will allocate some memory for the program name, which will be
+	 * free'd by an atexit() function.  The non-daemonized fork of this
+	 * program will quit without calling that function, which might cause
```

Comment:
s/quit/exit/

Reply:
Hmm.  I was trying to avoid using the same name as a library function, since the non-daemonized fork calls `_exit` rather than `exit`.

What about this instead:

> The non-daemonized fork of this program will call _exit() to quit without calling that function.  This might cause a memory checker to issue an invalid warning which should be suppressed.